include/igl/find_cross_field_singularities.cpp

```diff
@@ -48,12 +47,12 @@ IGL_INLINE void igl::find_cross_field_singularities(const Eigen::PlainObjectBase
           j=z;
       assert(j!=-1);
 
-      missmatch+=Handle_MMatch(VF[vid][i],j);
+      mismatch+=Handle_MMatch(VF[vid][i],j);
     }
-    missmatch=missmatch%4;
+    mismatch=mismatch%4;
```

Comment:
Spaces: `mismatch += Handle_MMatch(...)`

Reply:
I would add them personally. But as @jdumas it is like that in many places. I think it should be possible to write a shell script to fix such issue globally. I will try to write such a thing once I will have a bit of time.